Makefile

```diff
@@ -224,12 +273,17 @@ ifndef ignore-not-found
 endif
 
 .PHONY: prepare
-prepare: manifests kustomize manager-kustomization
+prepare: projectfile manifests kustomize manager-kustomization
+
+.PHONY: projectfile
+projectfile: PROJECT.in
+	cp PROJECT.in PROJECT
+	sed -i 's/projectName: .*/projectName: $(KUBEBUILDER_PROJECT_NAME)/' PROJECT
```

Comment:
wonder if we would keep `opendatahub-operator` as the `projectName` and then using kustomize to actually amen the name where it make sense so we could maybe avoid having duplicated rbac, manager & co files

Reply:
Figured out a solution to this: `--project`. Also FYI, I had to rebase.